tests/cases/global2.py

```diff
@@ -0,0 +1,18 @@
+code_0 = 0
+code_1 = 1
+
+code_a = "a"
+code_b = "b"
+
+l_b = {
+    code_a,
+}
+
+l_c = {
+    code_b: code_0,
+}
+
+if __name__ == "__main__":
+    for i in l_b:
```

Comment:
This has been broken by ea70634e - the `for` on sets no longer works in rust.
`if "a" in l_b:` still works, and could be used instead here, but that is broken in dart (and all other languages other than rust per https://github.com/adsharma/py2many/issues/93).
If possible, I'd like to keep the test working with a second language, and getting `in` comparison working on dart is one way of doing it.